std/process.d

```diff
@@ -724,10 +724,10 @@ private bool isExecutable(in char[] path) @trusted nothrow @nogc //TODO: @safe
 version (Posix) unittest
 {
     import std.algorithm;
-    auto unamePath = searchPathFor("uname");
-    assert (!unamePath.empty);
-    assert (unamePath[0] == '/');
-    assert (unamePath.endsWith("uname"));
+    auto lsPath = searchPathFor("ls");
+    assert (!lsPath.empty);
+    assert (lsPath[0] == '/');
+    assert (lsPath.endsWith("ls"));
```

Comment:
`ls` comes on every Android device and likely every other POSIX-ish system, while `uname` isn't on Android.